src/site/content/en/mini-apps/mini-app-alternative-runtime-environments/index.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: post
+title: Other mini app runtime environments
+authors:
+  - thomassteiner
+date:  2021-02-05
+# updated: 2021-01-25
+description: |
+  This chapter presents a number of runtime environments for mini apps that are not mobile devices.
+tags:
+  - mini-apps
+---
+
+## More than just on mobile
+
+Apart from mobile devices, where mini apps are omnipresent and which are their natural habitat, mini
+apps have started to conquer other runtime environments like cars and the classic desktop.
+
```

Comment:
If you haven't read the rest of the series, someone might imagine that mini apps are being used globally. But the reality is that it's mostly China, right? So maybe they've "conquered" China (that's a strong phrase that you might want to avoid) but they haven't conquered the globe yet, right?

Reply:
I have made clear that their success is regionally limited.